test.sh

```diff
@@ -18,7 +18,7 @@ for x in Dockerfile*;do
   echo "======================="
   docker build -t runtimetestimage -f "${x}" .
   #passes an env variable of size more than 4K chars
-  time dgoss run -v `pwd`/test_scripts:/test_scripts -e TEST_LONG_VAR="${largevar}" runtimetestimage || ((i++))
+  time dgoss run -v `pwd`/test_scripts:/test_scripts -e TEST_LONG_VAR="${largevar}" -e 1VAR_NAME=varnameStartWithNumber -e var.name=varnameHasPeriod -e var-name=varnameHasHyphen -e _=varnameHasJustUnderscore runtimetestimage || ((i++))
```

Comment:
Now that I have seen this link, think I will get rid of underscore variable. I just used it as an edge test case for the this regex `VALID_ENVNAME_REGEX="^[[:alpha:]_][[:alnum:]_]*$"`
WDYT?